paimon-core/pom.xml

```diff
@@ -284,13 +284,23 @@ under the License.
                             <artifactSet>
                                 <includes combine.children="append">
                                     <include>com.squareup.okhttp3:okhttp</include>
+                                    <include>com.squareup.okio:okio-jvm</include>
+                                    <include>org.jetbrains.kotlin:kotlin-stdlib</include>
```

Comment:
Could we use kotlin-stdlib-jdk8?

Reply:
It cant be `kotlin-stdlib-jdk8`, because its empty.